client/src/views/HallView.tsx

```diff
@@ -50,7 +50,7 @@ const HallView = () => {
         brutes: data.brutes.map((b) => (b.name === brute.name ? {
           ...b, favorite: !wasFavorite,
         } : b)).sort((a, b) => (a.favorite === b.favorite
-          ? (a.id - b.id)
+          ? a.createdAt.getTime() - b.createdAt.getTime()
```

Comment:
I think there's an error in the createdAt typing, as I'm sure it's a string, not a date, and this will break.

Might be something to fix in the types

Reply:
It's a date in the backend, but it becomes a string in the front end

Reply:
There is no interface Brute on the frontend it's straight from the prisma model  :
![image](https://github.com/Zenoo/labrute/assets/29907681/423e7773-bd60-45c2-910d-9b74f99719c6)
In the code there are others time you use moment.utc on  createdAt but MomentIput take string and date as parameters 
![image](https://github.com/Zenoo/labrute/assets/29907681/be0cbc48-41ee-4fca-ba76-96b509a18f88)
I Might need a lil a bit of help here ^^

Reply:
Got it I made the changes by using moment too

Reply:
Perfect